src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -105,23 +105,41 @@ public function add(array $headers)
      * @param string|null $default The default value
      * @param bool        $first   Whether to return the first value or all header values
      *
-     * @return string|string[]|null The first header value or default value if $first is true, an array of values otherwise
+     * @return string|string[]|int|null The first header value or default value if $first is true, an array of values otherwise
```

Comment:
where does `int` come from? Tend to believe this case should be deprecated if valid

Reply:
It cames from using header that returns integer values, I don't know if we can/could deprecate that because it means transforming them to string (since we don't use strict_types it does the transformation when adding the return type)

Reply:
How can a header be an integer?

Reply:
It's in the test, we use integer everywhere for header like "Content-Length" maybe some of the ->assertSame are wrong ?

Reply:
the `int` would violate the phpdoc of `getValues`. getValues says it returns an array of strings and `get` suddenly returns an int although it uses `getValues` internally.